src/site/content/en/blog/samesite-cookie-recipes/index.md

```diff
@@ -0,0 +1,324 @@
+  `<iframes>`
+
+Cookies may be used here to personalize content if the current user has an
+existing account on the remote site, maintain a session state, store general
+preferences, enable usage statistics, and more.
+
+<figure class="w-figure  w-figure--center">
+  <img src="iframe.png"
+      alt="Diagram of a browser window where the URL of embedded content does
+        not match the URL of the page."
+      style="max-width: 35vw;">
+  <figcaption class="w-figcaption">
+    If the embedded content doesn't come from the same site as the top-level
+    browsing context, it's third-party content.
+  </figcaption>
+</figure>
+
+Additionally, as the web is inherently composable, `<iframes>` are used to embed
+content that is also viewed in a top-level or first-party context. Any cookies
+used by that site will be considered as third-party cookies when the site is
+displayed within the frame. If you're creating sites that you intend to be
+easily composed that also rely on cookies to function, you will also need to
+ensure those are marked for cross-site usage or that you can gracefully fallback
+without them.
+
+### "Unsafe" requests across sites
+
+While "unsafe" may sound slighly concerning here, this refers to any request
+that may be intended to change state and on the web that's primarily POST
+requests. Cookies marked as `SameSite=Lax` will be sent on safe top-level
+navigations, e.g. clicking a link to go to a different site. However something
+like a `<form>` submission via POST to a different site would not include
+cookies.
+
+<figure class="w-figure  w-figure--center">
+  <img src="safe-navigation.png"
+      alt="Diagram of a request moving from one page to another."
+      style="max-width: 35vw;">
+  <figcaption class="w-figcaption">
+    If the incoming request uses a "safe" method then the cookies will be sent.
+  </figcaption>
+</figure>
+
+This pattern is used for sites that may redirect the user out to a remote
+service to perform some operation before returning, for example redirecting to a
+third-party identity provider. Before the user leaves the site, a cookie is set
+containing a single use token with the expectation that this token can be
+checked on the returning request to mitigate CSRF attacks. If that returning
+request comes via POST then it will be necessary to mark the cookies as
+`SameSite=None; Secure`.
+
+### Remote resources
+
+Any remote resource on a page may be relying on cookies to be sent with that
+request, from `<img>` tags, `<script>` tags, and so on. Common use cases here
+include tracking pixels and personalizing content.
+
+This also applies to
+[`fetch` requests](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#Sending_a_request_with_credentials_included)
+from the page. If you see the `credentials: 'include'` option in the
+configuration this is a good indication that cookies may well be expected on
```

Comment:
"If you see the `credentials: 'include'` option in the configuration..."

Change to:

"If `fetch()` is called with the `credentials: 'include'` option..."

It states what you're referring to more clearly.

Reply:
Replaced.